main/ui/src/main/java/org/cryptomator/ui/controllers/ChangePasswordController.java

```diff
@@ -122,6 +149,51 @@ private void didClickChangePasswordButton(ActionEvent event) {
 		}
 	}
 
+	// ****************************************
+	// Password strength management
+	// ****************************************
+
+	private void checkPasswordStrength(String password) {
+		int strengthPercentage = 0;
```

Comment:
I would prefer to show a word rather than a percentage. A word like "weak" or "strong" is more descriptive than a percentage, which suggests a pretty accurate rating while there are just 5 gradings.

So we need `initialize.messageLabel.passwordStrength=Password Strength: %s` as well as `initialize.messageLabel.passwordStrength.0=Weak` etc.